requirements/edx/pre.txt

```diff
@@ -4,6 +4,9 @@
 #   * @mollydb - to check licensing
 #   * One of @e0d, @jarv, or @feanil - to check system requirements
 
+# Use a modern setuptools instead of distribute
+setuptools==9.1
```

Comment:
I'm curious why you're specifically pinning `setuptools` at version 9.1, when [the latest stable version is 14.3.1](https://pypi.python.org/pypi/setuptools). Why not use the latest stable version?